Bonobo.Git.Server/Git/GitLfs/LfsAppDataStorageProvider.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Web;
+
+namespace Bonobo.Git.Server.Git.GitLfs
+{
+    /// <summary>
+    /// Provides a simple LfsDataStorageProvider implementation where LFS objects are stored directly in the AppData folder of the Bonobo web site.
+    /// </summary>
+    public class LfsAppDataStorageProvider : ILfsDataStorageProvider
+    {
+        public string GetFileUrl(string urlScheme, string urlAuthority, string requestApplicationPath, string operation, string repositoryName, string oid, long size)
+        {
+            string url = string.Concat(
+                urlScheme,
+                "://",
+                urlAuthority,
+                "/",
+                repositoryName,
+                ".git",
+                requestApplicationPath,
+                "lfs/oid/",
+                oid);
+            return url;
+        }
+
+        private string DetermineAppDataPath()
+        {
+            string candidate1 = HttpContext.Current.Server.MapPath("~/App_Data");
```

Comment:
Does this storage provider allow for a path other than the App_Data folder? On our production version of Bonobo, the git repos are not stored in the app_data folder. There is a config setting called `RepositoryPath` that allows me to re-locate them.

Reply:
No.  It was my intention that we should start with the simplest working implementation and add on as needed.  There isn't much extra to add a configuration setting for the AppDataStorageProvider so I'll do that now.

Reply:
This has been upgraded to a FileSystemStorageProvider.  It will adhere to the optional setting from the web.config (added but commented out), or a path relative (../Lfs) to the repository path (default).  You can now point the the D: drive etc.